aptos-move/aptos-release-builder/data/upgrade-compiler-v2.yaml

```diff
@@ -0,0 +1,13 @@
+---
+remote_endpoint: ~
+name: "v1.23.1"
+proposals:
+  - name: proposal_1_upgrade_framework
+    metadata:
+      title: "Upgrade testnet to compiler V2"
+      description: "Upgrade testnet framework to use compiler v2"
+    execution_mode: MultiStep
+    update_sequence:
+      - Framework:
+          bytecode_version: 6
```

Comment:
should we use bytecode version 7 instead of 6 here? @sherry-x @vgao1996 @fEst1ck

Reply:
Are we ready to upgrade framework bytecode version to 7?

Reply:
it is already enabled on mainnet for non-aptos-framework code. If we want to use enum, we need version 7.